src/lib/getPkgVersion.ts

```diff
@@ -1,30 +1,54 @@
 // eslint-disable-next-line no-restricted-imports
 import fetch from 'node-fetch';
+import semver from 'semver';
 
 import pkg from '../../package.json';
 
 const registryUrl = 'https://registry.npmjs.com/rdme';
 
+/**
+ * @see {@link https://docs.npmjs.com/adding-dist-tags-to-packages}
+ */
+type npmDistTag = 'latest';
+
+/**
+ * Return the major Node.js version specified in our `package.json` config.
+ *
+ * @example 14
+ */
+export function getNodeVersion() {
+  const { node } = pkg.engines;
+  return semver.minVersion(node).major;
+}
+
 /**
  * The current `rdme` version
  *
  * @param npmDistTag the `npm` dist tag to retrieve. If this value is omitted,
  * the version from the `package.json` is returned.
  * @example "8.0.0"
- * @see {@link https://docs.npmjs.com/cli/dist-tag}
- * @note we mock this function in our snapshots
+ * @see {@link https://docs.npmjs.com/adding-dist-tags-to-packages}
+ * @note we mock this function in our snapshots, hence it's not the default
  * @see {@link https://stackoverflow.com/a/54245672}
  */
-export async function getPkgVersion(npmDistTag?: 'latest' | 'next'): Promise<string> {
+export async function getPkgVersion(npmDistTag?: npmDistTag): Promise<string> {
   if (npmDistTag) {
     return fetch(registryUrl)
       .then(res => res.json())
       .then(body => body['dist-tags'][npmDistTag])
-      .catch(() => {
+      .catch(err => {
         // eslint-disable-next-line no-console
-        console.error('error fetching version from npm registry');
+        console.error('error fetching version from npm registry:', err.message);
```

Comment:
should this use the logger?

Reply:
done: 1b6b85d9f31299b4b01cfd38612a27f976bd3ab4